### ISSUE TITLE
Temporary Splash Page

### ISSUE THREAD

**Issue by iansoper** [openventilatorregistry/Design#28]
Create a temporary splash page introducing the project.

It should contain
- Logo/Project Name
- Tagline for the project
- Elevator Pitch for businesses and hospitals
- Email Capture form - collect emails from visitors to let them know when the full site launches
- Social media links (Do we have any right now?)

Do we need anything else here?

**Comment by InnovativeInventor** [comment]
Is this a dupe/similar to #25?

**Comment by iansoper** [comment]
no, the landing page is the homepage for the app, this is a temporary page until the app is done

**Comment by jpoechill** [comment]
This was quick because what's on the current site is null. :(

![ovr-splash-01](https://user-images.githubusercontent.com/7801775/77611528-ca110a00-6ee2-11ea-9818-4993efe42b52.png)
